Resideo_eshopping/resideo_eshopping/lib/model/eshopping_model.dart

```diff
@@ -0,0 +1,103 @@
+class Product{
+
+  int _id;
+  String _title;
+  String _s_desc;
+  String _img;
+  int _price;
+  int _quantity;
+  String _l_desc;
+  String _category;
+  int _rating;
+  String _review;
+  
+  Product(this._title,this._s_desc,this._img,this._price,this._quantity,this._l_desc,this._category,this._rating,this._review);
+  Product.withId(this._id,this._title,this._s_desc,this._img,this._price,this._quantity,this._l_desc,this._category,this._rating,this._review);
+
+  int get id => _id;
+  String get title => _title;
+  String get s_desc => _s_desc;
+  String get img => _img;
+  int get price => _price;
+  int get quantity => _quantity;
+  String get l_desc => _l_desc;
+  String get category => _category;
+  int get rating => _rating;
+  String get review => _review;
+
+  set title(String newtitle)
+  {
+    if(newtitle.length > 0 && newtitle.length < 25)
```

Comment:
Validation of strings length is not a part of requirements in API creation. So, please remove all validations